projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -51,6 +52,10 @@
 - WordPress.com Block Editor: disable if all Jetpack blocks are disabled via a filter.
 - WordPress.com REST API: make sure post metadata is always an array.
 
+### Other
+- Admin Page: Do not show an error message with JSONP is disabled. The admin page does not use it.
+- Readme: update link to changelog file. No changelog entry necessary.
```

Comment:
Should we leave the "Other" section out of 9.6, since we're really only starting to populate it in 9.7?

```suggestion
```

Reply:
I'm fine with leaving it there, to be honest, since that's not very visible.